src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <!-- fonts -->
```

Comment:
Don't really need to add such comments.
Also, put link with stylesheet after linking fonts.